blockly-core/core/utils/xml.js

```diff
@@ -17,6 +17,12 @@
  * limitations under the License.
  */
 
+/* global Blockly, goog */
+
+// Ignore "Expected a conditional expression and instead saw an
+// assignment" warnings.
+/* jshint -W084 */
```

Comment:
I'm not sure whether to be proud or ashamed of this

`:%s/^\(\s*\)\(.*\); \(\w*\) = \(\w*\)\[\(.\)\];\(.*\)$/\1\2; \5 < \4.length;\6\r\1  \3 = \4[\5];/gc`